compiler/build.gradle.kts

```diff
@@ -3,6 +3,10 @@ import org.gradle.api.tasks.bundling.Jar
 import org.jetbrains.kotlin.gradle.dsl.KotlinCompile
 import org.jetbrains.kotlin.gradle.dsl.KotlinJvmCompile
 
+tasks.withType<Test> {
+    maxParallelForks = (Runtime.getRuntime().availableProcessors() / 2).takeIf { it > 0 } ?: 1
```

Comment:
@sfs Please use Math.max

Reply:
Sure. The code is from the Gradle documentation by the way (https://guides.gradle.org/performance/#parallel_test_execution).